plugins/inventory/ome_inventory.py

```diff
@@ -18,9 +18,58 @@
 short_description: Group inventory plugin on OpenManage Enterprise.
 description: This plugin allows to retrieve inventory hosts from groups on OpenManage Enterprise.
 version_added: "7.1.0"
-extends_documentation_fragment:
-  - dellemc.openmanage.ome_auth_options
 options:
+  hostname:
+    description:
+    - OpenManage Enterprise or OpenManage Enterprise Modular IP address or hostname.
+    - If the value is not specified in the task, the value of environment variable C(OME_HOST) will be used instead.
+    env:
+     - name: OME_HOST
```

Comment:
To keep it consistent with the parameter name, better to rename it as.
```suggestion
     - name: OME_HOSTNAME
```